ci-scripts/ci-runner.bash

```diff
@@ -194,9 +194,13 @@ else
     fi
 
     # Find modified or added user checks
+    if [[ $(hostname) =~ daint ]]; then
+        # Temporarily disable running modified checks on Daint
+        userchecks=""
+    fi
+
     userchecks=( $(git diff origin/master...HEAD --name-only --oneline --no-merges | \
```

Comment:
This needs to go to an else block

Reply:
I will move the assignment after the git commands, in order to avoid re-indentation.